### PR TITLE
merged 2 different files for quiz_results and syntax error on users.router

### PR DESCRIPTION
**Pull request by TITHI1244**
# Description

There were 2 different files for quiz_results controller and router. Also fixed some syntax errors on users.router.

Fixes # (issue)

# How to test?

npm run dev and go to /api/documentation

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by sandhana14 — APPROVED** [review]
Hi @Tithi.. Looks great

File: src/server/api/routes/users.router.js [hunk 1]
```diff
@@ -57,7 +58,9 @@ router.get('/:id', (req, res, next) => {
     .getUserById(req.params.id)
     .then((result) => res.json(result))
     .catch(next);
-
+});
+/**
+ * @swagger
```
Comment (cph-kiwi) [inline src/server/api/routes/users.router.js:63]: This has already been corrected in develop.

**Review by cph-kiwi — APPROVED** [review]
Just pull from origin and push tot he PR again. There might be some merge conflicts to resolve.

**Review by sofiiadidovych — APPROVED** [review]
Thank you, @TITHI1244! Eveverything looks good.

**Comment by TITHI1244** [comment]
Merged the conflicts with develop branch

**Pull request merged**